scripts/package-cli

```diff
@@ -7,7 +7,7 @@ cd $(dirname $0)/..
 
 GO=${GO-go}
 
-for i in crictl kubectl k3s-agent k3s-server k3s; do
+for i in crictl kubectl k3s-agent k3s-server k3s-etcdsnapshotk3s; do
```

Comment:
Do you need `k3s` here as well?

Reply:
No, it's calling back to the server binary.